node/http.ts

```diff
@@ -208,13 +209,15 @@ export class IncomingMessage extends NodeReadable {
 type ServerHandler = (req: IncomingMessage, res: ServerResponse) => void;
 
 export class Server extends EventEmitter {
-  #handler: ServerHandler;
   #listener?: Deno.Listener;
   #listening = false;
 
-  constructor(handler: ServerHandler) {
+  constructor(handler?: ServerHandler) {
     super();
```

Comment:
Because EventEmitter is no longer ES6 class after #1558, it seems that we also need to call `EventEmitter.call(this)` around here to make Server actually work as EventEmitter

Reply:
That's surprising - the same thing happens in `Process` class and it seems to work fine (no broken tests)

Reply:
Tried again, and the above doesn't happen anymore.. I might have been missing something.. 🤕

Reply:
Tangential: this probably needs to change anyway because in node you can do `const server = http.Server()` (plain function call, not a construct call.)